### PR TITLE
REST > getUserPayments / setup-delete payment methods doc + schemas

### PR DESCRIPTION
**Pull request by Ninjeneer**

### CODE REVIEW

File: src/assets/server/rest/v1/schemas/billing/billing-delete-payment-method.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "$id": "billing-delete-payment-method",
+  "title": "Delete user payment method Request",
+  "type": "object",
+  "properties": {
+    "userID": {
+      "$ref": "common#/definitions/id"
+    },
+    "paymentMethodId": {
+      "$ref": "common#/definitions/id"
```
Comment (ClaudeROSSI) [inline src/assets/server/rest/v1/schemas/billing/billing-delete-payment-method.json:10]: This is wrong.
This ID is in a format which is specific to the BILLING service provider ( i.e.: in a STRIPE-specific format).
Make sure to be more flexible here!
Suggestion:
      **"type": "string",
      "sanitize": "mongo"**

I assume we cannot remove "payment methods" anymore if we keep that definition!

File: src/assets/server/rest/v1/schemas/billing/billing-setup-payment-method.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  "title": "Setup user payment method Request",
+  "type": "object",
+  "properties": {
+    "userID": {
+      "$ref": "common#/definitions/id"
+    },
+    "paymentMethodId": {
+      "type": "string",
+      "sanitize": "mongo"
+    }
```
Comment (ClaudeROSSI) [inline src/assets/server/rest/v1/schemas/billing/billing-setup-payment-method.json:12]: This one is correct!

**Review by ClaudeROSSI — CHANGES_REQUESTED** [review]
small mistake on DELETE payment methods

File: src/assets/server/rest/v1/schemas/billing/billing-delete-payment-method.json (hunk 1 above)
Comment (Ninjeneer) [reply src/assets/server/rest/v1/schemas/billing/billing-delete-payment-method.json]: Indeed, fixed.

**Review by ClaudeROSSI — APPROVED** [review]
approved

**Pull request merged**